tests/phpunit/Utils/Connection/TestDatabaseTableBuilder.php

```diff
@@ -160,7 +160,12 @@ protected function generateListOfTables() {
 			__METHOD__
 		);
 
-		if ( $dbConnection->getType() === 'mysql' && method_exists( $dbConnection, 'listViews' ) ) {
+		// MW < 1.42
+		if (
```

Comment:
Don't we need a if clause for the other cases?

Reply:
What other cases do you mean? Other db types?

Under MW 1.42+, listTables does listViews job now.

Reply:
sorry, I missed some lines in the code. All good. Forget my comment :-)